test/base/TTokenRegistryInitializeTest.js

```diff
@@ -0,0 +1,89 @@
+// JS Libraries
+const withData = require('leche').withData;
+const { t, createMocks, NULL_ADDRESS } = require('../utils/consts');
+const { createTestSettingsInstance } = require("../utils/settings-helper");
+const LogicVersionsRegistryEncoder = require('../utils/encoders/LogicVersionsRegistryEncoder');
+
+// Mock Contracts
+const Mock = artifacts.require('./mock/util/Mock.sol');
+
+// Smart Contracts
+const TTokenRegistry = artifacts.require('./base/TTokenRegistry.sol');
+const Settings = artifacts.require("./base/Settings.sol");
+
+contract('TTokenRegistryInitializeTest', function (accounts) {
+    const logicVersionsRegistryEncoder = new LogicVersionsRegistryEncoder(web3);
+    let instance;
+    let settingsInstance;
+
+    beforeEach('Setup for each test', async () => {
+        const pauser = accounts[0];
+        
+        mocks = await createMocks(Mock, 10);
```

Comment:
I can't find where we are using this variable?

Reply:
unused, removed